src/translations/screens/FareContract.ts

```diff
@@ -168,7 +169,15 @@ const FareContractTexts = {
       case TransportMode.Tram:
         return _('trikk', 'tram');
       case TransportMode.Water:
-        return _('båt', 'boat');
+        if (subMode === 'highSpeedPassengerService') {
+          return _('hurtigbåt', 'speedboat');
```

Comment:
I don't think 'speedboat' is used anywhere else, so it might not be the right translation. @ckbilstad @mjenssen following today's discussion regarding "hurtigbåt", any thoughts on this? Potentially relevant that we will add "Ferry" as a separate section later, while these will remain in "hurtigbåt", so it could become a little bit confusing if there are ferries outside that section.

<img width="400" alt="boat_sections" src="https://github.com/AtB-AS/mittatb-app/assets/134292729/feb2098b-55bb-49d0-91d8-6ae066b835a7">

Reply:
Maybe something "passenger boat" is clearer? 

At least, if you google "speedboat", you won't get the type of boats we're talking about 😄

Reply:
Could be! Btw we should update Google Translate as well after the conclusion 😁 

<img width="266" alt="gtranslate" src="https://github.com/AtB-AS/mittatb-app/assets/134292729/d5b83c7f-6387-4a89-961b-a8fc02e4d153">

Reply:
Updated using "passenger boat" for now :+1: